common/client-sdk/wasm/service_client.go

```diff
@@ -0,0 +1,380 @@
+//go:build js && wasm
+// +build js,wasm
+
+package arkwasmclient
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"syscall/js"
+
+	"encoding/json"
+	"errors"
+
+	"github.com/ark-network/ark/common/client-sdk/rest/service/arkservicerestclient"
+	"github.com/ark-network/ark/common/client-sdk/rest/service/arkservicerestclient/ark_service"
+	"github.com/ark-network/ark/common/client-sdk/rest/service/models"
+	"github.com/go-openapi/runtime"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+)
+
+var (
+	invalidArgsError = errors.New("invalid number of arguments")
+)
+
+func main() {
+	js.Global().Set("RestServiceClient", RestServiceClientWrapper())
+	js.Global().Set("InvokeRestServiceClientMethod", RestServiceClientWrapper())
+	js.Global().Set("Log", Log())
+	select {}
+}
```

Comment:
Here we should expose things like Balance, Send etc..

Reply:
Done